assets/js/base/context/hooks/payment-methods/use-payment-method-interface.ts

```diff
@@ -124,38 +130,38 @@ export const usePaymentMethodInterface = (): Record< string, unknown > => {
 			isProcessing,
 		},
 		components: {
-			ValidationInputError,
+			LoadingMask,
 			PaymentMethodIcons,
 			PaymentMethodLabel,
-			LoadingMask,
+			ValidationInputError,
 		},
 		emitResponse: {
 			noticeContexts,
 			responseTypes,
```

Comment:
I see this TS warning
```
type 'typeof responseTypes' is not assignable to type 'responseTypes'.ts(2322)
payment-method-interface.ts(83, 2): The expected type comes from property 'responseTypes' which is declared here on type 'EmitResponseProps'
(property) EmitResponseProps.responseTypes: responseTypes
```
Might have something to do with enums, as they are not types. (only enum members can be types)

the same warning appears for `noticeContexts` after the typo I've mentioned in my other comment is fixed.